.env.example

```diff
@@ -0,0 +1,8 @@
+PORT=
+ES_HOST=
+MAX_NUMBER_OF_GF_SUGGESTIONS=
+GENES_SUGGESTIONS_INDEX_NAME=
+VARIANTS_SUGGESTIONS_INDEX_NAME=
+KEYCLOAK_URL=
```

Comment:
Since in master, we still use Ego, I would put EGO_API here instead of KEYCLOAK_URL :)